ETC.tsv

```diff
@@ -97,109 +97,109 @@ ETC_20150317_000096	$Although their bright color makes them pretty visible, it's
-ETC_20150317_000115	The leaves of this plant are stronger than leather.
+ETC_20150317_000115	$The leaves of this plant are stronger than leather.
 ETC_20150317_000116	$Red Rafflesia
 ETC_20150317_000117	$Raffly
-ETC_20150317_000118	Even after undergoing mutations Raffly continue to hunt small insects.
+ETC_20150317_000118	$Even after undergoing mutations Raffly continue to hunt small insects.
 ETC_20150317_000119	$Blue Raffly
 ETC_20150317_000120	$Gray Raffly
 ETC_20150317_000121	$Black Raffly
 ETC_20150317_000122	$Ridimed
 ETC_20150317_000123	$Blue Ridimed
-ETC_20150317_000124	Having evolved around water, its entire body is now made of solid wood.
+ETC_20150317_000124	$Even though it mutated from a herb, its entire body is now made of solid wood.
 ETC_20150317_000125	$Black Ridimed
 ETC_20150317_000126	$Red Ridimed
 ETC_20150317_000127	$Sakmoli
-ETC_20150317_000128	Attacking its legs will not cause it to topple over.
+ETC_20150317_000128	$Attacking its legs will not cause it to topple over.
 ETC_20150317_000129	$Green Sakmoli
 ETC_20150317_000130	$Orange Sakmoli
 ETC_20150317_000131	$Black Sakmoli
 ETC_20150317_000132	$Rikaus
 ETC_20150317_000133	$Green Apparition
-ETC_20150317_000134	Ghost affected by the demons tend to change their appearances like this.
+ETC_20150317_000134	$Ghosts affected by Demons often change their appearance in this way.
 ETC_20150317_000135	$Black Apparition
 ETC_20150317_000136	$Spectre
 ETC_20150317_000137	$Black Spectre
-ETC_20150317_000138	Unlike other monsters in Myan Leong Ryu, these monsters often were evil souls even when they were living.
+ETC_20150317_000138	$Unlike other wraith monsters, these monsters were often evil souls even when they were living.
 ETC_20150317_000139	$Giant Spectre
 ETC_20150317_000140	$Spectre of Deceit
 ETC_20150317_000141	$Green Rikaus
 ETC_20150317_000142	$Spectron
 ETC_20150317_000143	$Spectre Master
 ETC_20150317_000144	$Thornball
-ETC_20150317_000145	This floating seed has transmutated, losing its outer fluff for a solid shell.
+ETC_20150317_000145	$This floating seed has transmutated, losing its outer fluff for a solid shell and thorns. However, its floating capability have stayed intact.
 ETC_20150317_000146	$Big Thornball
 ETC_20150317_000147	$Woodspirit
 ETC_20150317_000148	$Blue Woodspirit
 ETC_20150317_000149	$Green Woodspirit
 ETC_20150317_000150	$Woodman
-ETC_20150317_000151	It would be a real blessing if the Woodman couldn't move.
+ETC_20150317_000151	$The fact that the Woodman can't move is a real blessing.
 ETC_20150317_000152	$Blue Woodman
 ETC_20150317_000153	$Gray Woodman
 ETC_20150317_000154	$Red Woodman
 ETC_20150317_000155	$Dark Slime
-ETC_20150317_000156	Its slimy substance allows it to model itself into a prototype of the object it last encountered, thus the slime can exist in a variety of types.
+ETC_20150317_000156	$Although slimes have similar appearance, they vary in species due to difference between the materials they originated from.
```

Comment:
Although slimes have similar appearance -> Although slimes are similar in appearance
difference -> differences